tests/neo.UnitTests/Ledger/UT_MemoryPool.cs

```diff
@@ -487,8 +487,14 @@ public void TestUpdatePoolForBlockPersisted()
             var key2 = CreateStorageKey(Prefix_FeePerByte);
             key1.ScriptHash = NativeContract.Policy.Hash;
             key2.ScriptHash = NativeContract.Policy.Hash;
-            snapshot.Storages.Add(key1, item1);
-            snapshot.Storages.Add(key2, item2);
+            snapshot.Storages.GetAndChange(key1, () => new StorageItem
```

Comment:
@ixje, this change is expected, right? Since the value was previously existing.

Reply:
I don't remember, but it is most likely caused by the TestBlockchain isolation problem. I think the best way forward is to have that fixed first, reset all changes to the unit tests and try again. Some of the changes (like here) have likely been applied because of the isolation problem and might not be needed anymore once that is fixed

Reply:
AHeauehuaea, @ixje, this change was done by me, please check the last commits.

The question was to be sure this was consistent.

All UTs were fixed, the PR is kind of ready to merge.